django/db/backends/oracle/base.py

```diff
@@ -399,6 +399,14 @@ def __init__(self, connection):
     def _output_number_converter(value):
         return decimal.Decimal(value) if '.' in value else int(value)
 
+    @staticmethod
+    def _get_decimal_converter(precision, scale):
```

Comment:
Maybe `_output_decimal_converter` to keep consistency of converters names.

Reply:
I tried to emphasize that it's not converter by itself, but returns converter.